app/(content)/drill/[id]/(segments)/index.js

```diff
@@ -32,7 +32,7 @@ export default function Index() {
 
     return (
         <PaperProvider>
-            <Appbar.Header statusBarHeight={0}>
+            <Appbar.Header statusBarHeight={30}>
```

Comment:
I changed this cause on iOS at least for me this button was way too close to the top of the screen and almost unclickable.